global_costmap: 
  global_frame: map
  robot_base_frame: base_link

  update_frequency: 10.0
  publish_frequency: 10.0
  transform_tolerance: 0.5

  static_map: true 
  rolling_window: true

  width: 16
  height: 16
  resolution: 0.05
